### PR TITLE
Weeky 2017 36

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Weekly, not weeky, push of RN and website updates

### Resolves:
 [OKTA-140171](https://oktainc.atlassian.net/browse/OKTA-140171)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Hold on, I think we have a fix from Bella needed in weekly.

**Comment by mystiberry-okta** [comment]
waiting for a revert of c56d1a6

File: _source/_code/php/quickstart-signin-widget.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: software
-title: PHP
+title: Okta Sign-In Widget
```
Comment (jmelberg-okta) [inline _source/_code/php/quickstart-signin-widget.md]: This needs to be changed back to `PHP`.

Currently, we parse the `title` attribute in our markdown files to highlight the correct language on the left nav.

Comment (mystiberry-okta) [reply _source/_code/php/quickstart-signin-widget.md]: Will fix. It's a conflict merge error, I think.

**Comment by mystiberry-okta** [comment]
closing this branch and starting over.  Since we already pulled one pre-mature commit, seems like the safest thing to do.

**Pull request closed without merging**